<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="stockList">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>771</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Watchlist</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stockToFind">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>121</y>
      <width>680</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>120</y>
      <width>80</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="ticker">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>168</y>
      <width>148</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Ticker</string>
    </property>
   </widget>
   <widget class="QLabel" name="company">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>168</y>
      <width>147</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Company Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>170</y>
      <width>147</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
